admin/README.md

```diff
@@ -1,48 +1,14 @@
 # Admin workflow
-To kick off a conference, the OpenReview superuser needs to:
+To kick off a conference, the OpenReview superuser should:
 - Create a conference directory under openreview-scripts/venues
-- Create and post the conference's root group to the system
-- Create the conference administrator account (Optional)
+- Copy the file openreview-scripts/admin/conference-template/params.data and fill it in with conference-specific variables
```

Comment:
copy and paste it in which place? is there a default location for the params.data?

Reply:
I guess I'm imagining that users will create a directory based on the group's ID, but it should work even if it's in a different place

e.g. they'll create the directory venues/TestConf.org/2018/Conference, and then put the params file there.

I'll update the readme to say that